presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -231,6 +231,16 @@ Property Name                                        Description
 
 ``hive.metastore.glue.default-warehouse-dir``        Hive Glue metastore default warehouse directory
 
+``hive.metastore.glue.aws-access-key``               AWS access key to use to connect to the Glue Catalog. If
```

Comment:
Looks like this ended up here twice

Reply:
No, one is access-key and the other is for secret-key. They look similar.

Reply:
I am pretty sure I am looking at both of them twice? There are 4 doc entries here and both secret key and access key are here twice. Or maybe I am going cross eyed :D

Reply:
I double checked, just one copy :)

Reply:
OK, I know why I am being confused, one of them is the property name and the each property refers to the other property and the way it lined up in GH it looked like multiple instances. LOL.